CMake/AddDependencies.cmake

```diff
@@ -0,0 +1,83 @@
+# Set options for SQLiteCpp, disable all their tests and lints:
```

Comment:
Is this file a straight copy-paste?

Reply:
Change: https://github.com/cuberite/cuberite/pull/4743#discussion_r426781751